frontend/src/components/MgmtGameList.tsx

```diff
@@ -0,0 +1,269 @@
+import type React from "react";
+import {
+  Text,
+  Container,
+  Group,
+  NativeSelect,
+  Box,
+  useMantineTheme,
+  Pagination,
+  Flex,
+  Center,
+  Card,
+  Modal,
+  Button,
+} from "@mantine/core";
+import type { Game } from "../data/game";
+import { useState, useMemo } from "react";
+import { GAME_LIST_PAGE_LENGTH } from "../global.config";
+import MgmtEditGame from "./MgmtEditGame";
+
+export interface MgmtGameDisplayProps {
+  game: Game;
+  /**
+   * Callback when the user select the game to edit
+   * @param gameId game id of the selected game
+   * @returns none
+   */
+  onEdit: (gameId: string) => void;
+  /**
+   * Callback when the user select the game to remove
+   * @param gameId game id of the selected game
+   * @returns none
+   */
+  onRemove: (gameId: string) => void;
+}
+
+/**
+ * A component to display individual item.
+ */
+const MgmtGameDisplay: React.FC<MgmtGameDisplayProps> = ({
+  game: { id: gameId, name, exchangeRate },
+  onEdit,
+  onRemove,
+}) => {
+  return (
+    <>
+      {/* The display card */}
+      <Card shadow="md" padding="md" radius="md" h="100%" withBorder>
+        <Flex
+          mt="md"
+          gap="md"
+          direction="column"
+          wrap="nowrap"
+          h="calc(100% - 200px)"
```

Comment:
Since this works, we should leave the code as it is... But this height calculation is originally for cards with picture (which are 200px) so that the card can scale its own height based on the neighboring cards. It should be broken here, and yet it's not. My jet-lagged brain can't figure out why.

Reply:
I have no idea how CSS works..